platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -97,7 +97,7 @@ index 979352299..3771d19c9 100644
      private static class Stub implements IStub {
          private IDownloaderService mItf = null;
 -        final Messenger mMessenger = new Messenger(new Handler() {
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        private final MessengerHandlerServer mMsgHandler = new MessengerHandlerServer(this);
```

Comment:
The line 101-103 maybe from a different branch, please validate this is required for rebranding

Reply:
checked! this is same as the original